.github/workflows/publish.yaml

```diff
@@ -0,0 +1,35 @@
+name: Publish Python Package
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
```

Comment:
is there a problem with going with 3.12?

Reply:
No problem. Updated